test/libraries/SwapMath.t.sol

```diff
@@ -208,8 +213,14 @@ contract SwapMathTest is Test, GasSnapshot {
 
         // didn't reach price target, entire amount must be consumed
         if (sqrtQ != sqrtPriceTargetRaw) {
+            uint256 absAmtRemaining;
+            if (amountRemaining == type(int256).min) {
+                absAmtRemaining = uint256(type(int256).max) + 1;
+            } else if (amountRemaining < 0) {
+                absAmtRemaining = uint256(-amountRemaining);
+            }
```

Comment:
does it not need an `else absAmtRemaining = amountRemaining` ?

Reply:
yes! its because type(int256).min and type(int256).max do not have the same absolute value

Reply:
so i had to convert it to uint256 first then add 1

Reply:
but itll still do the else statement so its more clear!